### ISSUE TITLE
extensions.privateTab.fixAfterTabsButtonsAccessibility doesn't work in Firefox 40+

### ISSUE THREAD

**Issue by Infocatcher** [Infocatcher/Private_Tab#180]
Icons on buttons after the last tab don't grow with our paddings (and images disappear).

**Comment by Dimas-sc** [comment]
I've the same problem in Firefox 40 (Beta) and 41 (Aurora)? http://i.imgur.com/bqZI79P.png

**Comment by Infocatcher** [comment]
Yes, that's it.
Could you please try this test version, that includes required fixes: https://github.com/Infocatcher/Private_Tab/issues/177#issuecomment-120065324

**Comment by Dimas-sc** [comment]
It works! Thx :)
